scripts/variantstore/wdl/GvsCreateVATFromAnnotations.wdl

```diff
@@ -0,0 +1,549 @@
+              v.gnomad_eas_ac,
+              v.gnomad_eas_an,
+              v.gnomad_eas_af,
+              v.gnomad_fin_ac,
+              v.gnomad_fin_an,
+              v.gnomad_fin_af,
+              v.gnomad_nfr_ac,
+              v.gnomad_nfr_an,
+              v.gnomad_nfr_af,
+              v.gnomad_sas_ac,
+              v.gnomad_sas_an,
+              v.gnomad_sas_af,
+              v.gnomad_oth_ac,
+              v.gnomad_oth_an,
+              v.gnomad_oth_af,
+              v.revel,
+              v.splice_ai_acceptor_gain_score,
+              v.splice_ai_acceptor_gain_distance,
+              v.splice_ai_acceptor_loss_score,
+              v.splice_ai_acceptor_loss_distance,
+              v.splice_ai_donor_gain_score,
+              v.splice_ai_donor_gain_distance,
+              v.splice_ai_donor_loss_score,
+              v.splice_ai_donor_loss_distance,
+              g.omim_phenotypes_id,
+              g.omim_phenotypes_name,
+              v.clinvar_classification,
+              v.clinvar_last_updated,
+              v.clinvar_phenotype,
+              FROM `~{dataset_name}.~{variant_transcript_table}` as v
+              left join
+              (SELECT gene_symbol, ANY_VALUE(gene_omim_id) AS gene_omim_id, ANY_VALUE(omim_phenotypes_id) AS omim_phenotypes_id, ANY_VALUE(omim_phenotypes_name) AS omim_phenotypes_name FROM `~{dataset_name}.~{genes_table}` group by gene_symbol) as g
+              on v.gene_symbol = g.gene_symbol'
+
+  >>>
+    # ------------------------------------------------
+    # Runtime settings:
+    runtime {
+        docker: "openbridge/ob_google-bigquery:latest"
+        memory: "3 GB"
+        preemptible: 3
+        cpu: "1"
+        disks: "local-disk 100 HDD"
+    }
+    # ------------------------------------------------
+    # Outputs:
+    output {
+        Boolean done = true
+    }
+}
+
+task BigQueryExportVat {
+    input {
+        String contig
+        String project_id
+        String dataset_name
+        String output_path
+        String table_suffix
+        String? service_account_json_path
+        String load_jsons_done
+    }
+
+    String vat_table = "vat_" + table_suffix
+    String export_path = output_path + "export/" + contig + "/*.tsv.gz"
+
+    String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
+
+    command <<<
+
+        echo "project_id = ~{project_id}" > ~/.bigqueryrc
+
+        if [ ~{has_service_account_file} = 'true' ]; then
+          gsutil cp ~{service_account_json_path} local.service_account.json
+          export GOOGLE_APPLICATION_CREDENTIALS=local.service_account.json
+          gcloud auth activate-service-account --key-file=local.service_account.json
+          gcloud config set project ~{project_id}
+        fi
+
+        # note: tab delimiter and compression creates tsv.gz files
+        bq query --nouse_legacy_sql --project_id=~{project_id} \
+        'EXPORT DATA OPTIONS(
+        uri="~{export_path}",
```

Comment:
this is going straight to GCS?

Reply:
yes---and we'll need to create a new step that merges them all

Reply:
(the specs have changed, so it looks like we wont need to loop over chrs either)